### PR TITLE
Feature/cc 0120 cc app my order page

### PR DESCRIPTION
**Pull request by GLC-coder**
Feature cc 0120 cc-app my order page -ssr is ready be merged into main branch
*******Please make sure you have installed the stripe and run stripe on your terminal *****
Screen Shot 2022-11-14 at 2 17 17 pm
Screen Shot 2022-11-14 at 2 17 39 pm
Screen Shot 2022-11-14 at 2 17 49 pm

***Please make sure you have stripe configuration in your backend repo***
image

### CODE REVIEW

File: src/components/NavBar/index.tsx [hunk 1]
```diff
@@ -114,6 +116,7 @@ const NavigationBar = () => {
   };
 
   const handleCreateTemplateOpen = () => {
+    router.push("/");
```
Comment (STY1997) [inline src/components/NavBar/index.tsx:119]: why need to redirect to the same page?

**Review by STY1997 — COMMENTED** [review]
TQL

**Comment by ShawnWang86b** [comment]
image

**Comment by ShawnWang86b** [comment]
image

File: src/components/MyOrder/MyOrderList.tsx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+                {order.paidAt ? format(parseISO(order.createdAt), "dd/mm/yyyy hh:mm") : ""}
+              </Text>
+            </Flex>
+          </Flex>
+          <Flex width="5%" justifyContent="flex-end">
+            <Flex flexDirection="column">
+              <Text fontSize="12px" fontStyle="italic" fontWeight="300">
+                Status
+              </Text>
+              <Text color={order.status === "completed" ? "fontcolor.green" : "fontcolor.red"}>
```
Comment (Double-Twelve) [inline src/components/MyOrder/MyOrderList.tsx]: fontSize should be 20px

**Review by wwwseven — APPROVED** [review]

**Review by Cecilia0109 — APPROVED** [review]
LGTM

**Pull request merged**